platform/binance/api.go

```diff
@@ -109,6 +108,16 @@ func NormalizeTx(srcTx *Tx, token string) (tx blockatlas.Tx, ok bool) {
 	return tx, false
 }
 
+func filterTx(src []Tx, token string, txType string) []Tx {
+	var result []Tx
+	for _, tx := range src {
+		if tx.Asset == token && tx.Type == txType {
+			result = append(result, tx)
```

Comment:
Can you add this function: https://github.com/thoas/go-funk#funkfilter

Reply:
let's use this lib going forward to optimize all maps, fliter, sort and etc functionality